src/modules/settings/AppSettings.js

```diff
@@ -81,9 +85,22 @@ class AppSettings {
     await this.asyncStorage.removeItem(SELECTED_CITY_KEY)
   }
 
+  setIntroShown = async () => {
+    await this.asyncStorage.setItem(INTRO_SHOWN_KEY, JSON.stringify(true))
+  }
+
+  loadIntroShown = async (): Promise<boolean> => {
+    const value = await this.asyncStorage.getItem(INTRO_SHOWN_KEY)
+    if (value === null && process.env.E2E_TEST_IDS) {
+      return true
+    }
+    return value ? JSON.parse(value) : false
+  }
+
   clearAppSettings = async () => {
     await this.asyncStorage.removeItem(CONTENT_LANGUAGE_KEY)
     await this.asyncStorage.removeItem(SELECTED_CITY_KEY)
+    await this.asyncStorage.removeItem(INTRO_SHOWN_KEY)
```

Comment:
Not sure whether this should be removed on `clearAppSettings`, what do you think?

Reply:
Probably the same for the language (or at least we have to set a new language afterwards)